### PR TITLE
Format JS and CSS

### PR DESCRIPTION
**Pull request by tehp**
Uses [js-beautify](https://github.com/beautify-web/js-beautify)

Helps a lot with code readability (for me at least).

### CODE REVIEW

**Comment by Continuities** [comment]
Whoa, that's a huge diff.

**Comment by tehp** [comment]
Yeah 😐 
Nothing much you can do about that.

File: script/dropbox.js [hunk 1]
```diff
@@ -1,362 +1,384 @@
...
+            if (client.isAuthenticated()) {
 
-    /**
-     * Initiate dropbox connection
-     *
-     * @param interactive
-     * @param callback
-     */
-    connectToDropbox: function (interactive, callback) {
+                var datastoreManager = client.getDatastoreManager();
```
Comment (SubSage) [inline script/dropbox.js]: Someone else sign off on this code please, I haven't played with Dropbox stuff myself too much

**Comment by SubSage** [comment]
Deleted comment, someone just check the whole dropbox segment please.

**Comment by groteworld** [comment]
Given this changeset's size, I think the only way I can faithfully merge this would be to run your commands myself. if everything matches, and I can play the game, I'll merge. Otherwise, I'll stick my changes under your commit, It's not really a trust issue, it's more that this is so big, I'd never be 100% sure whats in the repo, and what we are supporting.

@tehp can you comment the commands you ran?

**Comment by Continuities** [comment]
I agree with @groteworld. A diff this large is hard to parse with the brainmeats.

**Comment by tehp** [comment]
Makes sense, I probably wouldn't trust myself either.

Anyways, I use Atom + [atom-beautify](https://atom.io/packages/atom-beautify) 

But if you aren't an Atom user you can just use [js-beautify](https://github.com/beautify-web/js-beautify), as this is the beautifier that I was using to format both CSS and JS.

[Here](https://github.com/beautify-web/js-beautify#python) is how to get started 😃 

Good luck, hopefully you don't run into any issues.
